Boost/BoostConfig.cmake

```diff
@@ -0,0 +1,27 @@
+#
```

Comment:
I don't think neither `BoostConfig.cmake` nor `CURLConfig.cmake` should be needed here, as they are only needed for finding the libraries binaries (i.e. from the client's build), not for building each library (which is done via the scripts).

Did you actually needed them? Or it was just because copying all the files from client's repo?

Reply:
yeah, I can remove them; I wasn't sure if it's better to just track the whole build output from `hubstaff-client`'s `third_party` here, or if we should be manually filtering out only the lib files. It seems simpler to just track the whole output rather than requiring devs to filter out some files each time they update the libs. But I have no strong preferences on the matter.

Reply:
Not sure what do you mean about filtering out....
This files are _only_ needed wherever the library must be found (i.e. where it is used). Here they are not needed at all.

The scripts are different. They are going to be removed from main client's repo, so they will exist only where the library is built. But that is part of a separate initiative (the one about boost update Michal is doing).

So for now, I would leave the clutter as little as possible, so this files should not be here. This way, the other initiative will be simpler, as less things to consider